### DISCUSSION TITLE
add small snippet for architectures

### DISCUSSION THREAD

**Discussion by talhaanwarch** [Project-MONAI/MONAI#2619] [Q&A]
I am checking `DynUNet` and i am confused what values should I used for kernel_size, strides and  upsample_kernel_size . can you add a small snippet with random values going on model and initialize model and output the shape.
For example in case of unet , the snippet would be like

```
x=torch.randn((1,1,96,96,96))
model = UNet(
                      dimensions=3,
                      in_channels=1,
                      out_channels=5,
                      channels=(16, 32, 64, 128, 256),
                      strides=(2, 2, 2, 2),
                      num_res_units=2,
                      norm=Norm.BATCH,
                  )
out=model(x)
out.shape
```

**Comment by talhaanwarch** [comment]
@Nic-Ma

**Reply by Nic-Ma** [reply]
Hi @talhaanwarch ,

You can take a look at the unit tests of DynUNet, it has rich examples for different inputs and args and outputs:
https://github.com/Project-MONAI/MONAI/blob/dev/tests/test_dynunet.py

Thanks.
